internal/manifest/validate_load_balancer.go

```diff
@@ -53,6 +65,16 @@ func (l *LoadBalancer) Validate(m *Manifest) error {
 			if _, ok := roles[role]; !ok {
 				return fmt.Errorf("role %q used inside cluster %q is not defined", role, cluster.Name)
 			}
+
+			if role == apiServerRole {
+				// check if this is an ApiServer LB and another ApiServer LB already exists.
+				if apiServerLBExists[cluster.TargetedK8s] {
+					return fmt.Errorf("role %q is used across multiple Load-Balancers for k8s-cluster %s. Can have only one ApiServer Load-Balancer per k8s-cluster", role, cluster.TargetedK8s)
```

Comment:
@Despire Can we make this clear in the docs as well? we should let users know about this limitation

Reply:
Yes.